packages/protocol/contracts/L1/libs/LibProposing.sol

```diff
@@ -43,16 +49,43 @@ library LibProposing {
             msg.sender != resolver.resolve("solo_proposer", false)
         ) revert L1_NOT_SOLO_PROPOSER();
 
-        if (txList.length > config.maxBytesPerTxList) revert L1_TX_LIST();
+        if (input.txListByteEnd <= input.txListByteStart)
```

Comment:
```suggestion
        if (input.txListByteEnd < input.txListByteStart)
```
I guess making an empty range still valid to allow creating empty blocks could still be useful to bring the L1 data to L2 for example.